### PR TITLE
fix bug: AiMessage text content is not copied when toolCalls are pres…

### PR DESCRIPTION
**Pull request by hrhrng**
## Issue

https://github.com/langchain4j/langchain4j/issues/986

## Change

When OpenAI returns both content and tool_calls, keep them all instead of just keeping the tool_calls.

## General checklist

- [x] There are no breaking changes
- [x] I have added unit and integration tests for my change
- [x] I have manually run all the unit and integration tests in the module I have added/changed, and they are all green
- [x] I have manually run all the unit and integration tests in the [core](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-core) and [main](https://github.com/langchain4j/langchain4j/tree/main/langchain4j) modules, and they are all green
- [x] I have added/updated the [documentation

### CODE REVIEW

**Comment by langchain4j** [comment]
HI @hrhrng, thank you! I wil ltry to review it asap

File: langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/InternalOpenAiHelper.java
Comment (Wadimz) [inline langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/InternalOpenAiHelper.java]: maybe we can add new static constructor method `aiMessage(String, List)` to follow the same style as in previous line?

Comment (hrhrng) [reply langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/InternalOpenAiHelper.java]: I have add static constructor, please take a review.

**Review by Wadimz — APPROVED** [review]

**Comment by hrhrng** [comment]
@Wadimz Hi, this is my first pull request, I wonder is there anything else I should do to make this request merged?

**Comment by Wadimz** [comment]
Hi @hrhrng! Congratulations on your first pull request! 
You'll need workflow approval for the pipeline run and a review from one of the repository owners for this pull request. Getting a review from other contributors can help speed up the process. Typically, @langchain4j reviews the changes and merges them once they've been approved. Don't worry if it takes some time. I suggest keeping your pull request up to date with the main repository to be ready for merge when reviewed.

**Comment by hrhrng** [comment]
> from

Thanks for your answer🙏.

**Comment by hrhrng** [comment]
> Hi @hrhrng! Congratulations on your first pull request! You'll need workflow approval for the pipeline run and a review from one of the repository owners for this pull request. Getting a review from other contributors can help speed up the process. Typically, @langchain4j reviews the changes and merges them once they've been approved. Don't worry if it takes some time. I suggest keeping your pull request up to date with the main repository to be ready for merge when reviewed.

Thanks for your answer🙏. @Wadimz

**Review by ghost — APPROVED** [review]
@hrhrng thanks a lot, good job!

The same change can be applied to `InternalAzureOpenAiHelper`, would you like to open a PR for this? 🙏

**Comment by hrhrng** [comment]
> @hrhrng thanks a lot, good job!
> 
> The same change can be applied to `InternalAzureOpenAiHelper`, would you like to open a PR for this? 🙏

@langchain4j 👌，I‘d like to fix it.

**Comment by hrhrng** [comment]
> @hrhrng thanks a lot, good job!
> 
> The same change can be applied to `InternalAzureOpenAiHelper`, would you like to open a PR for this? 🙏

@langchain4j Hi, I've pull request for this issue in https://github.com/langchain4j/langchain4j/pull/1576, please have a check.

**Pull request merged**
